core/src/main/scala/com/evolutiongaming/catshelper/Serial.scala

```diff
@@ -45,10 +46,10 @@ object Serial {
                 .void
             }
 
-            Concurrent[F].uncancelable {
+            Concurrent[F].uncancelable { _ =>
               for {
-                d <- Deferred.uncancelable[F, Either[Throwable, A]]
```

Comment:
we might need to create own `uncancelable` it depends on usages.

Reply:
As I was told by TL folks at discord channel, their advice is to use `deferred.get.uncancelable` explicitly in the usage-site. I think that makes sense, will rework it here.

Reply:
done